guides/source/active_record_querying.md

```diff
@@ -2511,6 +2511,13 @@ and may yield this for PostgreSQL:
 
 [`explain`]: https://api.rubyonrails.org/classes/ActiveRecord/Relation.html#method-i-explain
 
+### Explain options
+For databases and adapters which support them (currently PostgreSQL and MySQL), options can be passed to provide deeper analysis:
+
+```ruby
+Customer.where(id: 1).joins(:orders).explain(:analyze, :verbose)
```

Comment:
Maybe an example of the query being executed here would be helpful?

Just to set the expectation of the reader. 🙏

Reply:
Added